thorlcr/graph/thgraphmaster.cpp

```diff
@@ -674,19 +674,21 @@ cost_type CMasterActivity::calcFileReadCostStats(bool updateFileProps)
 
         return readCost;
     };
-
-    cost_type totalReadCost = 0;
+    cost_type readCost = 0;
+    ThorActivityKind actKind = container.getKind();
+    bool bIndexReadActivity = isIndexReadActivity(actKind);
```

Comment:
we don't normally use Hungarian naming convention

Reply:
I don't normally but I couldn't come with better name given that isIndexReadActivity() is already in use.